crates/router/src/connector/payme/transformers.rs

```diff
@@ -76,9 +102,27 @@ pub struct GenerateSaleRequest {
     seller_payme_id: Secret<String>,
     sale_callback_url: String,
     sale_payment_method: SalePaymentMethod,
+    services: Option<ThreeDS>,
     language: String,
 }
 
+#[derive(Debug, Serialize)]
+pub struct ThreeDS {
```

Comment:
Nit(naming convention): `ThreeDs`

Reply:
Raising a seperate PR to resolve this comments👍